schedule/__init__.py

```diff
@@ -44,30 +44,26 @@
 import random
 import re
 import time
+from typing import Set, List, Optional, Callable, Union
 
-logger = logging.getLogger('schedule')
+logger = logging.getLogger("schedule")
+TagType = str
```

Comment:
Just out of curiosity, what is the rationale for creating a type alias?

Reply:
I was uncertain if `str` is actually the proper type. Could the tags be also integers? Maybe also tuples?

Reply:
Apparently [tags must be hashable](https://github.com/dbader/schedule/blob/32a14006dc3aa875bd37c1aeb65de7b526941961/schedule/__init__.py#L409-L410). I guess we could use [the corresponding type](https://stackoverflow.com/a/38078003/2328729)?

Reply:
Nice, I didn't know that existed! I made use of it now.